nixos/modules/services/printing/cupsd.nix

```diff
@@ -296,11 +296,16 @@ in
     # gets loaded, and then cups cannot access the printers.
     boot.blacklistedKernelModules = [ "usblp" ];
 
+    # Some programs like print-manager rely on this value to get
+    # printer test pages.
+    environment.sessionVariables.CUPS_DATADIR = "${bindir}/share/cups";
+
     systemd.packages = [ cups.out ];
 
     systemd.sockets.cups = mkIf cfg.startWhenNeeded {
       wantedBy = [ "sockets.target" ];
-      listenStreams = map (x: replaceStrings ["localhost"] ["127.0.0.1"] (removePrefix "*:" x)) cfg.listenAddresses;
+      listenStreams = [ "/var/run/cups/cups.sock" ]
```

Comment:
Don't we want this to be `/run/cups/cups.sock`
* https://github.com/NixOS/nixpkgs/commit/0cf1944c36de87a3fb231adec090bea595092f94

also noticing that there's
https://github.com/NixOS/nixpkgs/blob/f3282c8d1e0ce6ba5d9f6aeddcfad51d879c7a4a/pkgs/misc/cups/default.nix#L44
in the package

Reply:
Ah ok, can update this. The reason this worked for me is my original branch was off of 19.03, but did not catch this change.